docs/topics/components-stability.md

```diff
@@ -59,37 +59,37 @@ We make sure to document precisely which subcomponents are not stable. We also d
+| Kotlin/JVM                                            | Stable       | 1.0                      |                                                                                   |
+| Kotlin K2 (JVM)                                       | Alpha        | 1.7                      |                                                                                   |
+| kotlin-stdlib (JVM)                                   | Stable       | 1.0                      |                                                                                   |
+| Coroutines                                            | Stable       | 1.3                      |                                                                                   |
+| kotlin-reflect (JVM)                                  | Beta         | 1.0                      |                                                                                   |
+| Kotlin/JS (Classic back-end)                          | Stable       | 1.3                      | Deprecated from 1.8.0, read the [IR migration guide](js-ir-migration.md)          |
+| Kotlin/JVM (IR-based)                                 | Stable       | 1.5                      |                                                                                   |
+| Kotlin/JS (IR-based)                                  | Stable       | 1.8                      |                                                                                   |
+| Kotlin/Native Runtime                                 | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin/Native memory manager                          | Stable       | 1.9.20                   |                                                                                   |
+| klib binaries                                         | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin Multiplatform                                  | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin/Native interop with C and Objective C          | Beta         | 1.3                      |                                                                                   |
+| CocoaPods integration                                 | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin Multiplatform Mobile plugin for Android Studio | Beta         | 0.8.0                    | [Versioned separately from the language](multiplatform-mobile-plugin-releases.md) |
+| expect/actual language feature                        | Stable       | 1.9.20                   |                                                                                   |
```

Comment:
Only top-level functions/properties are stable. Expect/actual classes still in Beta

Reply:
Added a separate row for classes